packages/js-sdk/src/user/user.ts

```diff
@@ -247,4 +247,29 @@ export class User {
 
     return this;
   }
+
+  async invalidateTokens() {
+    if (!this.isActive()) {
+      return this;
+    }
+
+    this.unregisterFromLiveService();
```

Comment:
The whole method seems to match the logout and is not trivial, better to reuse it and just extract the try/catch block to be passed as parameter.